src/sql/workbench/services/objectExplorer/browser/media/objectTypes/objecttypes.css

```diff
@@ -9,6 +9,12 @@
 	background: url("Table.svg") center center no-repeat;
 }
 
+.vs .codicon.table,
```

Comment:
>codicon [](start = 5, length = 7)

Hm, why is this change necessary? I'd think we'd be using the same classes for displaying the action icons. And Kevin fixed a bunch of issues a while back where we had to update the icons to use codicon instead so I feel like this is just a case of these being missed...

Reply:
Some things use .icon, and other use .codicon. Our dialogs use .codicon, but the table icon was only defined for .icon before.

Reply:
Yes, but we really should align to use codicon everywhere. We had icon because that's what vs code originally had but they changed everything over to codicon a while back and to be consistent I think it'd be easiest to just also make sure we're using codicon everywhere.

There's an argument to be made for making an adsicon or something to differentiate our icons vs theirs - but honestly I think that's just adding extra confusion for no real gain.

Anyways that's all outside the scope of this PR - my original point was why is this necessary for this PR? I'm guessing it's because the action is adding the codicon class to the class list but not the icon class?

Reply:
Items in the object explorer use .icon, for example.

Reply:
This is a styling change to align the chart toolbar with the design mockups. The show table button needed a table icon, but we use .codicon for buttons in dialogs.